app/controllers/admin/affiliates_controller.rb

```diff
@@ -66,7 +66,38 @@ class Admin::AffiliatesController < Admin::AdminController
       config.columns[c].form_ui = :textarea
     end
 
-    update_columns = attribute_columns.reject { |column| column =~ /\A(api_access_key|created_at|external_css_url|favicon_url|has_staged_content|id|nutshell_id|theme|updated_at)\z/i }
+    update_columns = %i[
```

Comment:
I slightly refactored this in the process of removing the reference to `nutshell_id`. I realized that this method followed a pattern that had caused problems in the past, where the array of columns was constructed by rejecting specified columns. Similar code caused problems when new columns were added to various tables, and those columns showed up automatically in Super Admin. This change ensures that any columns required for affiliate updates are added intentionally rather than automatically.